vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -1,1724 +1,2933 @@
 <html layout:decorate="~{layouts/main_staff}">
-    <head>
-        <link rel="stylesheet"
-            href="https://unpkg.com/easymde/dist/easymde.min.css">
-        <script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
-        <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css" rel="stylesheet" />
-        <script src="//cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/js/select2.min.js"></script>
-		<script th:src="@{/resources/jquery/jquery-ui.min_custom_sortable.js}"></script>
-		<script src="//use.fontawesome.com/releases/v5.8.1/js/all.js"
-    	data-auto-replace-svg="nest"></script>
-        <style type="text/css">
-            .hova {
-            background-color: #f2f2f2;
-            }
-            .EasyMDEContainer .CodeMirror-wrap{
-            overflow-y: scroll;
-            height: 400px;
-            }
-            .fixed {
-		    position: fixed;
-		    top:0; left:0;
-		    width: 100%;
-		    z-index:100;
-     		}
-     		
-     		li.active>a{
-			background: solid var(--primary);
-			color: #fff;
-			font-size: 15px;
-			}
-			
-			a.nav-link.active.show {
-			border-color: transparent;
-			border-bottom: 4px solid var(--primary);
-			color: var(--primary) !important;
-			background: transparent;
-			}
-			
-			
-        </style>
-        <script th:inline="javascript">
-            //# sourceURL=click.js
-            var contentCount = [[${#lists.size(slideContents)}]];
-            var markDown = null;
-            var defaultValue;
-            var addVideoClk=0;
-            var editVideoClk=0;
+   <head>
+      <link rel="stylesheet" href="https://unpkg.com/easymde/dist/easymde.min.css">
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+      <script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
+      <link href="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/css/select2.min.css" rel="stylesheet" />
+      <script src="//cdnjs.cloudflare.com/ajax/libs/select2/4.0.6-rc.0/js/select2.min.js"></script>
+      <script th:src="@{/resources/jquery/jquery-ui.min_custom_sortable.js}"></script>
+      <script src="//use.fontawesome.com/releases/v5.8.1/js/all.js" data-auto-replace-svg="nest"></script>
+      <style type="text/css">
+         .hova {
+         background-color: #f2f2f2;
+         }
+         .EasyMDEContainer .CodeMirror-wrap{
+         overflow-y: scroll;
+         height: 400px;
+         }
+         .fixed {
+         position: fixed;
+         top:0; left:0;
+         width: 100%;
+         z-index:100;
+         }
+         li.active>a{
+         background: solid var(--primary);
+         color: #fff;
+         font-size: 15px;
+         }
+         a.nav-link.active.show {
+         border-color: transparent;
+         border-bottom: 4px solid var(--primary);
+         color: var(--primary) !important;
+         background: transparent;
+         }
```

Comment:
where did all the indentation go? and did anything change here? please don't reformat or I can' see what changed.